website/src/main/tut/docs/rules/FixSyntax.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: docs
+title: FixSyntax
+---
+
+# FixSyntax
```

Comment:
I echo @gabro's comment that we should not distinguish rewrites and linters this much. Linters should be able to provide auto-fixes and rewrites should be able to lint. Would it make sense to bake these into DisableSyntax with `noFinalVal` and `noNonFinalCaseClass`?

Reply:
![image](https://user-images.githubusercontent.com/4629312/34958509-4fa1ad9e-fa43-11e7-8884-58656a902a14.png)
```scala
case ((((((((((((a, b), c), d), e), f), g), i), j), k), l), m), n) =>
            DisableSyntaxConfig(a, b, c, d, e, f, g, i, j, k, l, m, n)
```